internal/util/typeutil/type.go

```diff
@@ -49,3 +49,5 @@ const (
 )
 
 const InvalidUniqueID = UniqueID(-1)
+const InvalidIntPK = int64(-1)
```

Comment:
We did restrict that PK can not be nagative or ""?

Reply:
fixed